overview/content-types/commands.nix

```
@@ -9,10 +9,15 @@ let
   # TODO: wrap in submodule and make it render itself
   commands = {
```

How about the following set of changes (sorry github doesn't let me suggest across changed and unchanged lines):
```suggestion
  session = {
```